### DISCUSSION TITLE
Welcome to AstroPhot Discussions!

### DISCUSSION THREAD

**Discussion by ConnorStoneAstro** [Autostronomy/AstroPhot#27] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Sometimes a feature, refactor, or new step gets so large it doesn't fit very well as an issue. Here is the place to discuss such large scope ideas and get feedback before taking the time to dive into code! 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community!

**Comment by ranbir7** [comment]
Hey there
Ranbir here 
I'm from India and
I'm an undergrad student
Pursuing electrical engineering👨‍🔧

**Reply by ConnorStoneAstro** [reply]
Hi Ranbir! Thanks for joining the team! I really appreciate the help with AutoProf! This code is going to be very useful for the astronomy community!
